css/styles.css

```
@@ -315,6 +343,53 @@
 
   .hotel {
     display: grid;
+    padding: 30px 72px;
+    grid-template-columns: 180px 150px 160px 1fr;
+    grid-template-rows: 40px 40px 40px;
```

Строки лучше не фиксировать по высоте.
Используй для минимальной высоты свойство `min-content`. Так у тебя не будет строка "расползаться" и при этом если вдруг текста станет больше, строка расширится (а сейчас контент просто выпадет на другую строку).

При этом, одну из строк лучше тоже задать `1fr`. Потому что картинка будет выше их всех.
В данном случае пространство визуально скорее в середине, поэтому получится так:
```suggestion
    grid-template-rows: min-content 1fr min-content;
```